app-api/src/main/java/org/openmrs/eip/app/utils/CountDTO.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.eip.app.utils;
+
+public interface CountDTO {
```

Comment:
I believe this is only used in the web layer, and that's where it should be, exclude DTO suffix

Reply:
I use this interface to convert data from queries on SenderSyncMessageHistoryRepository.java